src/handlers/ens.ts

```diff
@@ -246,14 +252,12 @@ export const fetchSuggestions = async (
               );
               if (!!hasAvatar && profilesEnabled) {
                 try {
-                  const images = await fetchImages(domain.name);
-                  queryClient.setQueryData(
-                    ensProfileImagesQueryKey(domain.name),
-                    images
-                  );
+                  const avatar = await fetchENSAvatar(domain.name, {
+                    cacheFirst: true,
```

Comment:
`cacheFirst` means that it will use the avatar in local storage (if exists), otherwise it will perform network request